sdk/communication/azure-communication-email/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-email</artifactId>
-  <version>1.0.0-beta.3</version> <!-- {x-version-update;com.azure:azure-communication-email;current} -->
+  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-communication-email;current} -->
```

Comment:
The version should be updated in `eng/versioning/version_client.txt` too.

Reply:
This is not resolved yet. I don't see the updates to `eng/versioning/version_client.txt`.